### PR TITLE
feat(metadata): Add entry metadata to sdk result

### PR DESCRIPTION
**Pull request by davidfateh**
This adds metadata if it exists to the `sdk.entry` object.

### CODE REVIEW

File: lib/entry.ts [hunk 1]
```diff
@@ -19,6 +20,7 @@ export default function createEntry(channel, entryData, fieldInfo, createEntryFi
     fields: fieldInfo.reduce((acc, info) => {
       acc[info.id] = createEntryField(info)
       return acc
-    }, {})
+    }, {}),
+    ...(entryMetadata ? { metadata: entryMetadata } : {})
```
Comment (jelz) [inline lib/entry.ts:24]: How about always having empty `{}` metadata?

File: package-lock.json [hunk 2]
```diff
@@ -6017,8 +6017,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```
Comment (jelz) [inline package-lock.json:6020]: Can you do `nvm use` before `npm i` and see if we get these?

Comment (davidfateh) [reply package-lock.json]: Fixed.

**Comment by davidfateh** [comment]
> LGTM. Maybe we need a unit test for this?

Added the tests.

**Pull request merged**